README.md

```diff
@@ -1,21 +1,19 @@
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/esta/issues)
 ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/Specifyapp/parsers/CI)
-[![Coverage Status](https://coveralls.io/repos/github/Specifyapp/parsers/badge.svg?branch=master)](https://coveralls.io/github/Specifyapp/parsers?branch=master) 
+[![Coverage Status](https://coveralls.io/repos/github/Specifyapp/parsers/badge.svg?branch=master)](https://coveralls.io/github/Specifyapp/parsers?branch=master)
 [![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/Specifyapp/parsers.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Specifyapp/parsers/context:javascript)
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/Specifyapp/parsers.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Specifyapp/parsers/alerts/)
 <a href="https://github.com/Specifyapp/parsers/graphs/contributors">
-  <img src="https://img.shields.io/github/contributors/Specifyapp/parsers" alt="Contributors" />
+<img src="https://img.shields.io/github/contributors/Specifyapp/parsers" alt="Contributors" />
 </a>
-<!-- [![CI Actions Status](https://github.com/Specifyapp/parsers/workflows/CI/badge.svg)](https://github.com/Specifyapp/parsers/actions) -->
-
-
-
 
+<!-- [![CI Actions Status](https://github.com/Specifyapp/parsers/workflows/CI/badge.svg)](https://github.com/Specifyapp/parsers/actions) -->
 
 # Specify Parsers
 
 ## What is Specify?
-Specify helps you unify your brand identity by collecting, storing and distributing design tokens and assets—automatically. 
+
+Specify helps you unify your brand identity by collecting, storing and distributing design tokens and assets—automatically.
```

Comment:
Specify helps you unify your brand identity by collecting, storing and distributing design tokens and assets — automatically.

Specify is a design API facilitating the distribution of your design data within your design system.

![design-api-full-dark](https://user-images.githubusercontent.com/3913276/156588899-4a8e0183-f81e-40b2-98e9-ad518d8b6813.jpg)